config/routes.js

```diff
@@ -311,6 +311,9 @@ module.exports.routes = {
   'POST /api/v1/entrances/from-duplicate/:id':
     'v1/entrance-duplicate/create-from-duplicate',
 
+  // Historisation
+  'GET /api/v1/historisations/location/:id': 'v1/location/history',
```

Comment:
`historisations` doesn't exist in english. Even `historization` has another meaning than the french word "historisation".

Moreover, are we speaking about a resource (un "historique") ou un process ("l'historisation") ? Generally, REST is about resources : so I would choose `history`.

About the REST route, I think it's more relevant to use `locations/:id/history`: we are getting the history of a particular location.

`historisations/location/:id`, in REST language, means :
- get all the historisations (let's say 'histories' instead, see above comment about french <-> english translation) => ok fine, I have all my histories 🟢 
- get the "location" of all the "histories" => doesn't make sense, there is no direct link between "all the histories" and "one location" 🔴 
- get the id of this "history location" => previous step doesn't have sense, so this one neither

Reply:
@Clm-Roig @julien-vendran : This route is returning all the histories of a specific location ? Then if it is the case it should be `locations/:id/histories` no ?
And if you only want to get a specific history the route should be : 
`locations/:id/histories/:dateReviewed` (as dateReviewed is the second part of the primary key)

Reply:
I used the word `history` as the english for `historique`: we don't say "les historiques de la localisation'" but "l'historique de la localisation". 

Is "histories" appropriate in english ? It is a plural form of "historique" legit in english ? Or does "history" should be only singular like "historique" in french ?

Reply:
I agree for the dateReviewed, good point !

Reply:
Noted for the name of the route ! Same here, you need to validate, or not, the usage of `versioning` instead. 
About `locations/:id/histories/:dateReviewed`, in our case, we get all version of a location but we don't need details about this (We already have all information about each location). So, for the moment, we don't think that this route is usefull to us.

Reply:
ok ! and yes use snapshots instead of histories in the route too when needed ;)